### ISSUE TITLE
Fix Get-MgReportSkype duplicates

### ISSUE THREAD

**Issue by peombwa** [microsoftgraph/msgraph-sdk-powershell#628]
![image](https://user-images.githubusercontent.com/7061532/115085835-0df4f980-9ec0-11eb-98a4-3e2275542c57.png) 
[AB#9045](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/9045)

**Comment by peombwa** [comment]
Depends on #407

**Comment by peombwa** [comment]
The fix will need to come from AutoREST and is being tracked by https://github.com/Azure/autorest.powershell/issues/795. AutoREST.PowerShell is truncating everything after `*Skype` when an `operationId` has `*SkypeForBusiness*` in it.

**Comment by peombwa** [comment]
Duplicate of https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/912
